src/lib/MsDyn.Contrib.CloneFieldDefinitions/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyTitle("MsDyn.Contrib.CloneFieldDefinitions")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Florian Kroenert")]
+[assembly: AssemblyCompany("Florian Kroenert (Edited by Remon Kamel)")]
```

Comment:
While I appreciate your work and would like to mention you as contributor, that won't be inside the AssemblyInfo.
If you're fine with that, I'd add a contributors section to the ReadMe.